assets/js/components.js

```diff
@@ -47,6 +47,27 @@ jQuery(function () {
         $(td).html(`${rowData.attributes.github_enforce_admins_enabled}`)
       },
     },
+    {
+      data: 'attributes.environments',
+      createdCell: function (td, _cellData, rowData) {
+        const environments = rowData.attributes.environments
+        const prodEnvironments = environments ? environments.filter(env => env.name === 'prod') : []
```

Comment:
Should this filter on env.type?

Reply:
Not sure it is, the name can be anything I believe, it's just a coincidence at the moment that for many envs it matches the type